include/seqan/store/store_contig.h

```diff
@@ -192,13 +192,13 @@ struct ContigFile
     static const TId INVALID_ID;
 
     CharString        fileName;
-    AutoSeqFormat    format;
+    //AutoSeqFormat    format; // is deprecated
```

Comment:
commenting out and removing it has exactly the same effect :nerd_face: 
so please delete it!